contracts/Governance/Timelock.sol

```diff
@@ -164,6 +176,17 @@ contract Timelock is Ownable {
         }
     }
 
+    function getPendingTransactions() external view returns (PendingTransaction[] memory) {
```

Comment:
@tkeidar / @AndrzejGisWideLab - WDYT we'll need? Only a function to get the pending transactions altogether? Or maybe we also need a getter to the length of the set so we could get them one by one?

Reply:
in solidity we can do `queuedTransactionsParamsHashes.length()` maybe it's also possible on js

Reply:
I believe it's like an array. You don't have it unless you'll add a function that exposes it

Reply:
So the real question here is:
Can we get all the values of an enumerable set one by one (like we do with an array)?
And do we need a function to expose the length of the enumerable set (like we do with an array)?

@tkeidar and I talked about it, we do want these abilities as well.
@AndrzejGisWideLab let's add it?

Reply:
I tried to use EnumerableMap, but unfortunately it's not extendable :( I added the functions you wanted